examples/DotNetCore/ChatApp/ChatApp/Message.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Microsoft.Extensions.Configuration;
+
+namespace ChatApp
+{
+    internal class Message
+    {
+        [ConfigurationKeyName("role")]

```

Comment:
Since the property name is "Role" and the JSON property name is also "role", we may not need to have the `ConfigurationKeyName`, do we have it for consistency ?

Reply:
I would keep it for consistency.